package.json

```diff
@@ -46,13 +49,17 @@
     "mkdirp": "^0.5.1",
     "mocha": "^2.2.4",
     "mocha-jsdom": "^1.0.0",
+    "optimist": "^0.6.1",
     "protractor": "^3.0.0",
+    "q": "^1.4.1",
     "require-dir": "^0.3.0",
+    "selenium-webdriver": "^2.48.2",
```

Comment:
Maybe I'm interpreting the error above incorrectly, and it's actually `at Object.<anonymous> (/Users/[username]/repos/cfpb/cfgov-refresh/node_modules/protractor/lib/protractor.js:3:17)` that's looking for selenium-webdriver